### PR TITLE
Feature of basic && commands

### PR DESCRIPTION
**Pull request by HanJiyao**
Basic functionality only, currently excute all and returns the last feedback

### CODE REVIEW

**Review by sharpstorm — COMMENTED** [review]
Could you do some test cases for this?

File: src/test/java/seedu/contax/logic/parser/ChainCommandParseTest.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import seedu.contax.commons.core.index.Index;
+import seedu.contax.logic.commands.AddCommand;
+import seedu.contax.logic.commands.ChainCommand;
+import seedu.contax.logic.commands.Command;
+import seedu.contax.logic.commands.EditCommand;
+import seedu.contax.testutil.EditPersonDescriptorBuilder;
+import seedu.contax.testutil.PersonBuilder;
+
+
+public class ChainCommandParseTest {
```
Comment (sharpstorm) [inline src/test/java/seedu/contax/logic/parser/ChainCommandParseTest.java:34]: Please check this file name.
```suggestion
public class ChainCommandParserTest {
```

**Review by sharpstorm — CHANGES_REQUESTED** [review]
Small issue, but looks good otherwise

File: src/test/java/seedu/contax/logic/parser/ChainCommandParseTest.java (hunk 1 above)
Comment (HanJiyao) [reply src/test/java/seedu/contax/logic/parser/ChainCommandParseTest.java]: thx changed alr

**Review by sharpstorm — APPROVED** [review]
LGTM.

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm! thanks

**Pull request merged**